index.html

```diff
@@ -2694,12 +2711,196 @@ <h1>
         <h2>
 DID Resolution
         </h2>
+
+        <p>
+The <a>DID resolution</a> process takes a <a>DID</a> and resolution options as
+input, and produces a <a>DID document</a> and resolution metadata as output. The
+process utilizes the "Read" operation of the applicable <a>DID method</a>, as
+described in <a href="#read-verify"></a>. The <a>DID method</a>-specific details
+of how this process is accomplished is outside the scope of this specification.
+        </p>
+
+        <section>
+          <h3>
+Generic DID Resolution Algorithm
```

Comment:
Is there a particular reason why we need the term "Generic" here, would "DID Resolution Algorithm" suffice?

Reply:
@tplooker Actually, I am strong proponent of constantly reinforcing that the DID spec defines **generic** DID syntax and **generic** DID resolution because that makes it clear that it is up to DID method specifications to define **specific** DID syntax (their own method name and method-specific identifier) and **specific** DID resolution operations (how their specific DID method performs the CRUD operations). @rhiaro and I are trying to use the same "generic/specific" terminology in our rewrite of the frontmatter sections (PRs coming shortly).